airflow/www/templates/airflow/dags.html

```diff
@@ -308,7 +308,9 @@ <h2>{{ page_title }}</h2>
                     data-dag-id="{{ dag.dag_id }}"
                     data-summary="{{ dataset_triggered_next_run_info[dag.dag_id] }}"
                   >
-                    {{ dataset_triggered_next_run_info[dag.dag_id] }}
+                    {% with %}{% set ds_info = dataset_triggered_next_run_info[dag.dag_id] %}
```

Comment:
```suggestion
                    {% with ds_info = dataset_triggered_next_run_info[dag.dag_id] %}
```

Simpler?

Reply:
I think I was reading out of date docs when I wrote this. Thanks!